### DISCUSSION TITLE
优化异步函数请求性能 | LeeDaisen | 肯了个德的前端技术博客 | 个人日记 | 电台与web创意分享 | 编程、设计、交互、技术探索

### DISCUSSION THREAD

**Discussion by giscus** [admin8756/leelaa-blog-comments#23] [Announcements]
# 优化异步函数请求性能 | LeeDaisen | 肯了个德的前端技术博客 | 个人日记 | 电台与web创意分享 | 编程、设计、交互、技术探索

从重复调用到高效管理异步的请求，避免多次调用产生的性能问题

https://leelaa.cn/posts/agzcaa

**Comment by admin8756** [comment]
欢迎各位前端大手子，来提供更好的方案
